src/python/grpcio_reflection/grpc_reflection/v1alpha/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The gRPC Authors
+# Copyright 2022 The gRPC Authors
```

Comment:
Copyright line only need to record the year that this file is checked in. In this case, this file is not added by this PR, I would say keep the original version.